runatlantis.io/docs/terraform-versions.md

```diff
@@ -15,13 +15,17 @@ projects:
 See [atlantis.yaml Use Cases](repo-level-atlantis-yaml.html#terraform-versions) for more details.
 
 ## Via terraform config
-Alternatively, one can use the terraform configuration block's `required_version` key to specify an *exact* version:
+Alternatively, one can use the terraform configuration block's `required_version` key to [specify an exact version or version constraint](https://developer.hashicorp.com/terraform/language/expressions/version-constraints#version-constraint-syntax):
 ```tf
```

Comment:
Let's add some example of the version constraint syntax and the terraform result of them